### DISCUSSION TITLE
parent 的正式翻譯

### DISCUSSION THREAD

**Discussion by ken71301** [python/python-docs-zh-tw#734] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

parent

### 術語翻譯

父 / 上代

### 說明

描述繼承時常會需要翻譯，例如 parent class
翻譯為 父類別 或 上代類別 何者較好?

目前於wiki中並無定義，而教育部樂詞網的推薦翻譯為 [上代；父；母；親項](https://terms.naer.edu.tw/detail/0913217ddb5c7738fd613ee531d1d4a3/?startswith=zh&seq=1)

**Comment by mattwang44** [comment]
parent class 我習慣相對於 subclass（子類別）翻譯成「父類別」，但單看 `parent` 一詞可能和前後文關係蠻大的、可能比較難歸納出一個固定翻譯

**Comment by ken71301** [comment]
3/30：分兩種翻譯，若為類似` parent-class`，即翻譯為`父`類別，但若為統稱，例如`their parent`，其指其繼承的所有物件的統稱，即翻譯為`上代`
child同理為 `子` 或 `下代`
